tests/e2e/default/enketo/pregnancy.wdio-spec.js

```diff
@@ -99,7 +128,7 @@ describe('Pregnancy registration', () => {
   });
 
   it('Should verify that all tasks related with the high risk pregnancy were created', async () => {
-    const tasksTitles = ['Health facility ANC reminder', 'Danger sign follow up', 'Pregnancy home visit'];
+    const tasksTitles = ['Health facility ANC reminder', 'Danger sign follow up', 'Delivery'];
```

Comment:
This happened because I changed the expected delivery date, if it is in a month the program will automatically create some tasks, and one is a `pregnancy home visit`, but since I changed the `edd` to 8 days the program created a `delivery` task instead a `Pregnancy home visit`